packages/lesswrong/server/dialogues/cron.ts

```diff
@@ -15,6 +15,7 @@ addCronJob({
         notificationType: "newDialogueChecks",
         documentType: null,
         documentId: null,
+        extraData: {userId: user._id}, // passed for the AB test
```

Comment:
I think it'd be better to refactor the `notificationMessage` logic in `notificationCallbackHelpers` to pass through the `userId` we already have in that context to `getMessage` (via `GetMesageProps`), since that's a whole lot safer than using `extraData`

Reply:
agreed that would be neat, but also, trying to make this a quick and lightweight PR, and seems like I shouldn't need to deal with the trapcard of refactoring some pretty central (and sort of harder to test code) as part of it

Reply:
Fully general excuse :)  But if you plan on deleting this after letting the AB test run for a few days, then I don't care very much